### PR TITLE
Modification of the meta-files

### PR DESCRIPTION
**Pull request by Diego-ort**
Modification of the meta-files (setup.py, package.xml, setup.cfg, CMakeLists.txt) of the carla_manual_control and carla_ros_bridge modules for foward-backward compatibility in ROS1 and ROS2

### CODE REVIEW

File: carla_manual_control/setup.py [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
+    setup(
+        name=package_name,
+        version='0.0.0',
+        packages=['src/' + package_name],
+        data_files=[
+            ('share/ament_index/resource_index/packages',
+             ['resource/' + package_name]),
+            ('share/' + package_name, ['package.xml']),
+            ('share/' + package_name + '/config',
+             ['config/settings.yaml', 'config/sensors.json'])
```
Comment (arkadiy-telegin) [inline carla_manual_control/setup.py]: settings.yaml and sensors.json are carla_ego_vehicle config files, no such files in manual_control :)

File: carla_manual_control/setup.py [hunk 2]
```diff
@@ -1,13 +1,45 @@
...
+        ],
+        install_requires=['setuptools'],
+        zip_safe=True,
+        maintainer='CARLA Simulator Team',
+        maintainer_email='carla.simulator@gmail.com',
+        description='CARLA ego vehicle for ROS2 bridge',
```
Comment (arkadiy-telegin) [inline carla_manual_control/setup.py]: wrong description

File: carla_manual_control/setup.py (hunk 1 above)
Comment (Diego-ort) [reply carla_manual_control/setup.py]: Setup.py is corrected now :)

File: carla_manual_control/setup.py (hunk 2 above)
Comment (Diego-ort) [reply carla_manual_control/setup.py]: Description has also been corrected

File: carla_manual_control/package.xml [hunk 3]
```diff
@@ -5,14 +5,27 @@
...
-  <exec_depend>carla_msgs</exec_depend>
-  <exec_depend>sensor_msgs</exec_depend>
-  <exec_depend>std_msgs</exec_depend>
+
+  <!-- ROS 2 DEPENDENCIES-->
+  <depend condition="$ROS_VERSION == 2">rclpy</depend>
+  <test_depend condition="$ROS_VERSION == 2">ament_copyright</test_depend>
+  <test_depend condition="$ROS_VERSION == 2">ament_flake8</test_depend>
+  <test_depend condition="$ROS_VERSION == 2">ament_pep257</test_depend>
+  <test_depend condition="$ROS_VERSION == 2">python3-pytest</test_depend>
```
Comment (SergiGM93) [inline carla_manual_control/package.xml]: Do we need pytest? we do not have any test yet, right?
We are missing `carla_msgs`, `sensor_msgs` and `std_msgs` for the ros2 version. I am also not sure if we need any dependency for launch

File: carla_ros_bridge/package.xml [hunk 4]
```diff
@@ -1,32 +1,41 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>carla_ros_bridge</name>
-  <version>0.0.1</version>
+  <version>0.0.0</version>
```
Comment (FabianSchurig) [inline carla_ros_bridge/package.xml]: version mismatch

File: carla_ros_bridge/package.xml [hunk 5]
```diff
@@ -1,32 +1,41 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```
Comment (FabianSchurig) [inline carla_ros_bridge/package.xml]: we don't need that line or?

File: carla_ros_bridge/setup.py [hunk 6]
```diff
@@ -1,13 +1,44 @@
 """
 Setup for carla_ros_bridge
 """
+import os
+ROS_VERSION = int(os.environ['ROS_VERSION'])
```
Comment (FabianSchurig) [inline carla_ros_bridge/setup.py:5]: `.get()`

File: carla_manual_control/setup.py [hunk 7]
```diff
@@ -1,13 +1,43 @@
 """
 Setup for carla_manual_control
 """
+import os
+ROS_VERSION = int(os.environ['ROS_VERSION'])
```
Comment (FabianSchurig) [inline carla_manual_control/setup.py:5]: `.get()`

File: carla_ros_bridge/package.xml (hunk 5 above)
Comment (FabianSchurig) [reply carla_ros_bridge/package.xml]: 7c2ff7b

File: carla_ros_bridge/package.xml (hunk 4 above)
Comment (FabianSchurig) [reply carla_ros_bridge/package.xml]: 7c2ff7b

File: carla_ros_bridge/setup.py (hunk 6 above)
Comment (FabianSchurig) [reply carla_ros_bridge/setup.py:5]: a6beead

**Review by FabianSchurig — APPROVED** [review]

**Pull request merged**